README.md

```diff
@@ -158,30 +162,30 @@ Its also possible to ship the unleash instance around yourself, instead of using
 ```js
 const { Unleash } = require('unleash-client');
 
-const instance = new Unleash({
+const unleash = new Unleash({
     appName: 'my-app-name',
     url: 'http://unleash.herokuapp.com',
 });
 
-instance.on('ready', console.log.bind(console, 'ready'));
-// required error handling when using instance directly
-instance.on('error', console.error);
+unleash.on('ready', console.log.bind(console, 'ready'));
+// required error handling when using unleash directly
+unleash.on('error', console.error);
 ```
 
 ## Events
 
 The unleash instance object implements the EventEmitter class and **emits** the following events:
 
-| event      | payload                          | description                                                                                 |
-| ---------- | -------------------------------- | ------------------------------------------------------------------------------------------- |
-| ready      | -                                | is emitted once the fs-cache is ready. if no cache file exists it will still be emitted.    |
-| registered | -                                | is emitted after the app has been registered at the api server                              |
-| sent       | `object` data                    | key/value pair of delivered metrics                                                         |
-| count      | `string` name, `boolean` enabled | is emitted when a feature is evaluated                                                      |
-| warn       | `string` msg                     | is emitted on a warning                                                                     |
-| error      | `Error` err                      | is emitted on a error                                                                       |
-| unchanged  | -                                | is emitted each time the client gets new toggle state from server, but nothing has changed  |
-| changed    | `object` data                    | is emitted each time the client gets new toggle state from server and changes has been made |
+| event      | payload                          | description                                                                                                                                                                                                                                  |
+| ---------- | -------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ready      | -                                | is emitted once the fs-cache is ready. if no cache file exists it will still be emitted. The client is ready to use, but might not have synchronized with the Unleash API yet. This means the SDK still can operate on stale configurations. |
```

Comment:
does this mean it's `registered` we actually want to wait for if we want to wait for updated data?

Reply:
No, registered only means the SDK has sent meta info to the API. 
For that use case you would wait for the first "changed" event today. 

We plan to add another event to make this clearer: https://github.com/Unleash/unleash-client-node/issues/209

Reply:
ah perfect, thanks!